angular-viewer/adam-angular-demo/src/app/components/object-results/object-results.component.ts

```diff
@@ -26,6 +27,20 @@ export class ObjectResultsComponent implements OnChanges {
       features.push(feat);
     });
 
+    const affordances: Affordances[] = [];
+    if (entry.affordances) {
+      entry.affordances.forEach((element) => {
+        const element_split = element.split("_")
+        let feat = {name: element};
+        if (element_split.length == 4) {
+          feat = {
+            name: `can be ${element_split[0]} in "${element_split.slice(1).join(" ")}"`
+          };
+        }
```

Comment:
The processing logic might not be strictly needed here given it's already there in the output generation from #1141. Maybe after that is merged it is worth testing to see if we need this.

Reply:
Right, that's what I figured as well after looking at the code Jacob wrote up. I don't think it would hurt too much performance-wise to keep in, but we can still check after that is merged.

Reply:
I'm less concerned with performance than "is this going to make sense to a reader." If I were to find this logic in both places I'd probably be confused and suspect I was wrong about where the data was coming from, because why would it be processed like this twice?

The other option is to stick a comment in here saying _in effect_ "yes, we already did this on the backend, but just to be safe."